packages/storybook/src/nimble/checkbox/checkbox.stories.ts

```diff
@@ -29,13 +30,31 @@ const metadata: Meta<CheckboxArgs> = {
         </${checkboxTag}>
     `),
     argTypes: {
+        label: {
+            name: 'default',
```

Comment:
I agree it would be good to distinguish actual names of slots/attributes from just descriptions like this. I will tackle this in a follow up PR. I'm tracking it in a task list in #824.

My only other idea besides "(default)" is "Default slot", under the hope that the space in its name makes it obvious that it's not `slot="Default slot"`.